src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.csproj.CoreCompileInputs.cache

```diff
@@ -0,0 +1 @@
+8e52c0f87939158bd8f55e7bd1ff0daa727f5202e54abb64ec070cc578e498a8
```

Comment:
The obj directory can be added to .gitignore.

Reply:
Done